py-polars/tests/test_df.py

```diff
@@ -700,12 +700,14 @@ def test_to_json(df):
 
 
 def test_from_rows():
-    df = pl.from_rows([[1, 2, "foo"], [2, 3, "bar"]], column_name_mapping={1: "foo"})
+    df = pl.DataFrame.from_rows(
```

Comment:
This will also work as `pl.from_rows` right?

Reply:
Yes it will! I figured the `test_from_rows` function here should test the actual `from_rows` function that's attached to the DataFrame, not the `functions` version (since it's `test_frame.py`). The other `from_rows` function is tested earlier in the file, in `test_concat`.

Reply:
Great! Then I merge it in. Thanks again. :)